### PR TITLE
feat: Hangup call (AR-1256)

### PR DESCRIPTION
**Pull request by ohassine**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-1256" title="AR-1256" target="_blank">SubtaskAR-1256</a>  Update UI
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

New feature

### Solutions

We can now hang up a started call or an ongoing call

Needs releases with:

- [ ] GitHub link to other pull request

#### How to Test

see video

### Attachments (Optional)

https://user-images.githubusercontent.com/18124919/163389046-a6499d29-c78c-48fa-929e-2a434ac1d4f4.mov

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-512/1/) **failed**.

**Comment by AndroidBob** [comment]
Build [2](https://10.10.124.134/job/wire-android-reloaded/job/PR-512/2/) **failed**.

**Comment by AndroidBob** [comment]
Build [3](https://10.10.124.134/job/wire-android-reloaded/job/PR-512/3/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3097435)(3097435)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3097435)(3097435)-dev-release.apk)

**Review by saleniuk — APPROVED** [review]

File: app/src/main/kotlin/com/wire/android/ui/calling/OngoingCallViewModel.kt [hunk 1]
```diff
@@ -42,7 +46,20 @@ class OngoingCallViewModel @Inject constructor(
     }
 
     private suspend fun initiateCall() {
-            //TODO pass conversation type
-            startCall.invoke(conversationId)
+        //TODO pass conversation type
+        startCall.invoke(conversationId)
```
Comment (trOnk12) [inline app/src/main/kotlin/com/wire/android/ui/calling/OngoingCallViewModel.kt:50]: small thingy but to keep it consistent I think we do not use the .invoke() but just use the use case as this startCall(converstaionId)

File: app/src/main/kotlin/com/wire/android/ui/calling/OngoingCallViewModel.kt [hunk 2]
```diff
@@ -42,7 +46,20 @@ class OngoingCallViewModel @Inject constructor(
...
+    }
+
+    fun hangUpCall() {
+        viewModelScope.launch {
+            endCall.invoke(conversationId)
```
Comment (trOnk12) [inline app/src/main/kotlin/com/wire/android/ui/calling/OngoingCallViewModel.kt:55]: small thingy but to keep it consistent I think we do not use the .invoke() but just use the use case as this endCall(converstaionId)

**Review by trOnk12 — COMMENTED** [review]
good job <3 small comments about using the use case

**Comment by AndroidBob** [comment]
Build [5](https://10.10.124.134/job/wire-android-reloaded/job/PR-512/5/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3138945)(3138945)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3138945)(3138945)-dev-release.apk)

**Pull request merged**